cmd/cli/kubectl-kyverno/data/api-group-resources.json

```diff
@@ -1813,6 +1813,10 @@
         {
           "groupVersion": "flowcontrol.apiserver.k8s.io/v1",
           "version": "v1"
+        },
```

Comment:
This file shouldn't have changed, I will update the makefile to pop an ephemeral cluster instead of using the underlying one, sorry.

Reply:
removed the changes